CODEOWNERS

```diff
@@ -22,3 +22,4 @@
 # Only wrangler-admins are needed to approve releases
 # Product teams can request hold on release by reaching out to wrangler-admins
 **/CHANGELOG.md @cloudflare/wrangler-admins
+**/package.json @cloudflare/wrangler
```

Comment:
Should that be?
```suggestion
**/package.json @cloudflare/wrangler-admins
```

Reply:
@Skye-31 i considered that but then `package.json` changes in non-release PRs would require wrangler-admin team approval. i think the default of wrangler team approval would address the intent here

Reply:
Good point!

Reply:
😀